### PR TITLE
changes to make things work as expected

### PR DESCRIPTION
**Pull request by ziafazal**
@mattdrayer  @e-kolpakov these are changes I made to make things work as expected after rebase with master branch having notifications. @e-kolpakov can you check the custom/requirements since drag and drop xblock was not working for me?.

### CODE REVIEW

File: cms/envs/common.py [hunk 1]
```diff
@@ -877,3 +877,75 @@
...
+    'null': {
+        'class': 'edx_notifications.channels.null.NullNotificationChannel',
+        'options': {}
+    }
+}
+
+# list all of the mappings of notification types to channel
+NOTIFICATION_CHANNEL_PROVIDER_TYPE_MAPS = {
+    '*': 'durable',  # default global mapping
+}
```
Comment (e-kolpakov) [inline cms/envs/common.py]: @ziafazal nit: no newline at the end of file - pylint should have yelled at you :)

**Comment by e-kolpakov** [comment]
@ziafazal drag and drop works fine here. If you are importing it from somewhere else it might be the case. Also, this version uses quite outdated XBlock, specifically, it does not include latest serialization-deserialization fixes. I'm sending a PR with XBlock version bump soon (just after I ensure it at least pretends to work fine). Other than that - changes look good (at least they work as expected both in LMS, studio and Apros), but I haven't yet checked if something else needs updating. I'll keep you informed, but this PR is good to merge.

**Comment by mattdrayer** [comment]
@chrisndodge -- here are some more changes to review

**Comment by mattdrayer** [comment]
I took a look at this changeset -- things appear to be in order overall in LMS -- it's mostly config and Sass/CSS changes.  I ran the LMS python unit tests for api_manager and observed all tests passing.  Along the lines of what @e-kolpakov mentioned regarding pylint -- I ran `paver run_quality`, comparing the branch against the fork's master, and there are a number of violations we should consider addressing.  Let's be sure to address those issues before merging this PR into ziafazal/ibirch-rebase

**Comment by ziafazal** [comment]
@e-kolpakov publish bug in studio is fixed. Yeah looks to me it emerged after rebase.

**Comment by ziafazal** [comment]
@mattdrayer @e-kolpakov how i can see quality report for a particular branch?. right now running quality gives me huge set of files.

**Comment by e-kolpakov** [comment]
@ziafazal `diff-quality` normally compares working copy with origin/master, but [there is option to compare to other branch](https://github.com/Bachmann1234/diff-cover#compare-branch). YOu might want to use it.

**Comment by ziafazal** [comment]
@e-kolpakov Thanks this does the trick.
`paver run_quality --compare-branch=ziafazal/ibirch-rebase`

**Comment by ziafazal** [comment]
@e-kolpakov @mattdrayer fixed quality errors. It is now 100% for both pep8 and pylint.

**Comment by e-kolpakov** [comment]
@ziafazal do I need to give thumbs up? If I do, here you go :+1:

**Comment by mattdrayer** [comment]
:+1:  from me as well now that the quality violations have been addressed

**Pull request merged**